.github/workflows/validate-integration-files.yml

```
@@ -59,19 +59,16 @@ jobs:
       - name: Validate icons
         run: |
           S3_BUCKET="port-graphical-assets"
-          S3_ICONS_COLOR_URL="https://$S3_BUCKET.s3.amazonaws.com?prefix=icons/blueprintsColor/"
-          color_results=$(curl -s "$S3_ICONS_COLOR_URL" | grep -o '<Key>[^<]*</Key>' | sed 's/<Key>\([^<]*\)<\/Key>/\1/' | tr '[:upper:]' '[:lower:]')
-
-          S3_ICONS_URL="https://$S3_BUCKET.s3.amazonaws.com?prefix=icons/blueprints/"
-          colorless_results=$(curl -s "$S3_ICONS_URL" | grep -o '<Key>[^<]*</Key>' | sed 's/<Key>\([^<]*\)<\/Key>/\1/' | tr '[:upper:]' '[:lower:]')
+          S3_ICONS_URL="https://$S3_BUCKET.s3.amazonaws.com?prefix=icons/apps/"
+          results=$(curl -s "$S3_ICONS_URL" | grep -o '<Key>[^<]*</Key>' | sed 's/<Key>\([^<]*\)<\/Key>/\1/' | tr '[:upper:]' '[:lower:]')
 
           # All integration icons
           icons=($(find integrations -name 'spec.yaml' -not -path "**/.venv/*" | xargs -I {} yq -r '.icon' {} | tr '[:upper:]' '[:lower:]'))
           bp_icons=($(find integrations/** -name 'blueprints.json' -not -path "**/.venv/*" | xargs -I {} jq -r '.[] | .. | .icon?' {} | grep -v '^null$' | sort -u | tr '[:upper:]' '[:lower:]'))
           merged_set=($(printf "%s\n" "${icons[@]}" "${bp_icons[@]}" | sort -u))
 
           for icon in "${merged_set[@]}"; do
-            if [ $(echo "$color_results" | grep -c "$icon") -eq 0 ] && [ $(echo "$colorless_results" | grep -c "$icon") -eq 0 ]; then
+            if [ $(echo "$results" | grep -c "$icon") -eq 0 ] && [ $(echo "$colorless_results" | grep -c "$icon") -eq 0 ]; then
```

```suggestion
            if [ $(echo "${RESULTS}" | grep -c "$icon") -eq 0 ] && [ $(echo "${colorless_results}" | grep -c "${icon}") -eq 0 ]; then
```

and yes, ofc the other variables should be caps as well.